hub/core/storage/cachable.py

```diff
@@ -27,7 +27,7 @@ def __setstate__(self, state: Dict[str, Any]):
         self.__dict__.update(state)
 
     def tobytes(self) -> bytes:
-        return bytes(json.dumps(self.__getstate__()), "utf-8")
+        return bytes(json.dumps(self.__getstate__(), sort_keys=True, indent=4), "utf-8")
```

Comment:
can you just add a comment at the top of this function that says "sortkeys / indent make the json more human readable"

Reply:
Ok